base/applications/sndvol32/dialog.c

```diff
@@ -113,9 +113,9 @@ AddDialogControl(
         ClassName = (LPWSTR)Offset;
 
         /* adjust offset */
-        Offset += wcslen(ClassName) + 1;
+        Offset += wcslen(ClassName);
```

Comment:
I'd find it more straightforward to keep this and use `WindowName = (PWSTR)Offset` below, but this is okay too.